catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -143,20 +144,20 @@ protected Object executeWithSubject() throws Exception {
         final File outputFile = initOutputFile(output);
         if (outputFile.exists()) {
             printErrorMessage(String.format("File [%s] already exists!", outputFile.getPath()));
-            return null;
```

Comment:
I don't have other ideas for integrating with your unit tests, but I'm more concerned right now about the runtime impact.